components/nimbus/ios/Nimbus/NimbusCreate.swift

```diff
@@ -0,0 +1,86 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+private let EXPERIMENT_BUCKET_NAME = "main"
+private let EXPERIMENT_COLLECTION_NAME = "nimbus-mobile-experiments"
+
+private let LOG_TAG = "Nimbus.swift"
+private let logger = Logger(tag: LOG_TAG)
+
+public let defaultErrorReporter: NimbusErrorReporter = { err in
+    switch err {
+    case is LocalizedError:
+        let description = err.localizedDescription
+        logger.error("Nimbus error: \(description)")
+    default:
+        logger.error("Nimbus error: \(err)")
+    }
+}
+
+extension Nimbus {
+    /**
+     *  Create an instance of `Nimbus`.
+     *
+     * - Parameters:
+     *     - server: the server that experiments will be downloaded from
+     *     - appSettings: the name and channel for the app
+     *     - dbPath: the path on disk for the database
+     *     - enabled: intended for FeatureFlags. If false, then return a dummy `Nimbus` instance. Defaults to `true`.
+     *     - errorReporter: a closure capable of reporting errors. Defaults to using a logger.
+     */
+    public static func create(_ server: NimbusServerSettings?,
+                              appSettings: NimbusAppSettings,
+                              dbPath: String,
+                              enabled: Bool = true,
+                              errorReporter: @escaping NimbusErrorReporter = defaultErrorReporter) -> NimbusApi {
+        guard enabled else {
+            return NimbusDisabled()
+        }
+
+        let context = Nimbus.buildExperimentContext(appSettings)
+        let remoteSettings = server.map { server -> RemoteSettingsConfig in
+            let url = server.url.absoluteString
+            return RemoteSettingsConfig(
+                serverUrl: url,
+                bucketName: EXPERIMENT_BUCKET_NAME,
```

Comment:
Will the client ever need / want to change these values?

Reply:
Hmm. Not sure; I'm not sure when it would make sense, TBH.